indico/modules/events/registration/client/js/form/FormItem.jsx

```diff
@@ -150,6 +185,11 @@ export default function FormItem({
       </>
     ) : null;
 
+  if (!show && !setupMode) {
+    form.change(rest.htmlName, null);
```

Comment:
This does not work for all fields. For example a simple text field will break. Sending no value is fine, but sending null values isn't.

Luckily the fix seems to be easy (tested only w/ a text field so far):

```suggestion
    form.change(rest.htmlName, undefined); 
```

Reply:
Changed.